project/Dependencies.scala

```diff
@@ -87,7 +87,7 @@ object Dependencies {
   val scalaUri: ModuleID =        "io.lemonlabs"                  %% "scala-uri"            % "3.0.0"
   val scalatest: ModuleID =       "org.scalatest"                 %% "scalatest"            % "3.2.2" % "test"
   val mockito: ModuleID =         "org.scalatestplus"             %% "mockito-3-4"          % "3.2.2.0" % Test
-  val mockserverNetty: ModuleID = "org.mock-server"               % "mockserver-netty"      % "3.9.2" % "test"
+  val mockserverNetty: ModuleID = "org.mock-server"               % "mockserver-netty"      % "5.11.2" % "test"
```

Comment:
why did this have to change?

Reply:
The old version was unhappy with Java 11, upgrading was the easiest way to deal with that, as far as @MatthewBemis and I could see.